### PR TITLE
BaSP-TG-95: Projects redux - Get

### PR DESCRIPTION
**Pull request by GonzaloAsencio**
- Create constants to get projects
- Create thunk to get projects
- Create projects reducer
- Create projects reducer
- Connect list component with store to get projects

### CODE REVIEW

File: src/redux/rootReducer.js [hunk 1]
```diff
@@ -1,9 +1,11 @@
 import { combineReducers } from 'redux';
 
 import timeSheetsReducer from './timeSheets/reducer';
+import ProjectsReducer from './projects/reducer';
 
 const rootReducer = combineReducers({
-  timeSheets: timeSheetsReducer
+  timeSheets: timeSheetsReducer,
+  Projects: ProjectsReducer
```
Comment (Ignacio-Sormani) [inline src/redux/rootReducer.js:8]: Use camel case for imports and store modules

File: src/Components/Projects/index.js [hunk 2]
```diff
@@ -72,19 +67,38 @@ const Projects = () => {
     },
     { heading: 'Actions' }
   ];
+  if (error) {
+    return (
+      <section className={styles.container}>
```
Comment (Ignacio-Sormani) [inline src/Components/Projects/index.js:72]: Use shared info modal

File: src/redux/rootReducer.js (hunk 1 above)
Comment (GonzaloAsencio) [reply src/redux/rootReducer.js:8]: Done

File: src/Components/Projects/index.js (hunk 2 above)
Comment (GonzaloAsencio) [reply src/Components/Projects/index.js]: Done

File: src/Components/Projects/index.js [hunk 3]
```diff
@@ -72,19 +67,33 @@ const Projects = () => {
     },
     { heading: 'Actions' }
   ];
+  if (error) {
+    return setTypeModal('Error'), setTextModal('There was an error'), openModal();
+  }
```
Comment (MaicolFreire001) [inline src/Components/Projects/index.js:72]: This implementation doesn't work, this throws [Uncaught Invariant Violation: Too many re-renders.] when an error occurs. 
You can check the code in the PR #80 for a solution

**Review by MaicolFreire001 — CHANGES_REQUESTED** [review]

File: src/Components/Projects/index.js (hunk 3 above)
Comment (GonzaloAsencio) [reply src/Components/Projects/index.js]: Done

**Review by fede230298 — APPROVED** [review]

File: src/redux/rootReducer.js [hunk 4]
```diff
@@ -1,9 +1,11 @@
 import { combineReducers } from 'redux';
 
 import timeSheetsReducer from './timeSheets/reducer';
+import projectsReducer from './projects/reducer';
 
 const rootReducer = combineReducers({
-  timeSheets: timeSheetsReducer
+  timeSheets: timeSheetsReducer,
+  Projects: projectsReducer
```
Comment (Ignacio-Sormani) [inline src/redux/rootReducer.js:8]: Should be project to maintain the consistency

Comment (GonzaloAsencio) [reply src/redux/rootReducer.js:8]: That's weird, I had changed it before

Comment (GonzaloAsencio) [reply src/redux/rootReducer.js]: Done!

**Review by fede230298 — APPROVED** [review]

**Review by MaicolFreire001 — APPROVED** [review]

**Pull request merged**